_posts/2021-07-01-quarkus-2x-platform-quarkiverse-registry.adoc

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: 'The New Quarkus 2.x Platform Model and Ecosystem Support in the Dev Tools'
+date: 2021-07-01
+tags: release
+synopsis: 'The Quarkus 2.0.0.Final release introduces the new Quarkus platform model and the dev tools that make it easy to discover and work with the Quarkus extension ecosystem.'
+author: aloubyansky
+---
+
+= The Quarkus 2.x Platform Model
+
+The Quarkus 1.x platform was based on a single BOM - `io.quarkus:quarkus-universe-bom`, that would be imported by Quarkus applications. The universe BOM would include all the Quarkus
+platform extensions plus their dependencies. BOMs are meant to make it easy to align common dependencies on versions that work for all the libraries and frameworks used in an application.
+However, badly composed BOMs may actually make it very difficult or practically impossible to achieve that. Generally speaking, the wider the scope of the dependencies a BOM manages,
+the higher the risk for its users of running into incompatiblity issues including libraries that aren't managed by the BOM into their applications.
+
+The `io.quarkus:quarkus-universe-bom`, as its artifactId implies, is certainly a large and diverse BOM, including dependencies of Camel, Google Cloud Services, Kogito, OptaPlanner
+and other Quarkus platform members. Importing `io.quarkus:quarkus-universe-bom` will enforce ~3600 dependency version constraints on an application. Besides the fact that it's practically
+always way more than necessary, it may actually cause a serious compatibility issue that could have been avoided if the Quarkus platform BOM was not trying to manage the "universe".
+
+E.g. including a library that depends on a different version of the `commons-beanutils:commons-beanutils` than Camel Quarkus into an application that actually does not include any
+Camel Quarkus extension may become problematic, because the `io.quarkus:quarkus-universe-bom` will enforce the vesion required by Camel Quarkus.
```

Comment:
```suggestion
Camel Quarkus extension may become problematic, because the `io.quarkus:quarkus-universe-bom` will enforce the version required by Camel Quarkus.
```